build.gradle

```diff
@@ -35,6 +35,13 @@ import org.gradle.util.DistributionLocator
 import java.nio.file.Files
 import java.nio.file.Path
 import java.security.MessageDigest
+plugins {
+    id 'com.gradle.build-scan' version '1.13.2'
+}
+buildScan {
+    termsOfServiceUrl = 'https://gradle.com/terms-of-service'
+    termsOfServiceAgree = 'yes'
```

Comment:
I'm not sure we can speak for everyone who forks the project.

Reply:
It's unfortunate Gradle didn't implement a property for this, but I just realized that we can do something like `project.properties.get('org.elasticsearch.scanTermsOfServiceAgree', 'no')`.  Then one can add `org.elasticsearch.scanTermsOfServiceAgree=yes` to `~/.gradle/gradle.properties` to change that default. What do you think of that  ?

Reply:
I'm not 100% sure we need to do this at all - the instructions for newer versions of gradle don't mention this code and I'm not really sure what having `-scan` work for everyone buys us. But if we feel the need to do this then I figure we could use a system property. So if you run `./gradlew -DscanTOS=agree` then we set up scan. If you don't specify it, then we don't.

I'd honestly lean towards not doing this at all though. I think maybe if folks need to scan they can add it themselves?